bin/git-summary

```diff
@@ -22,6 +25,10 @@ for arg in "$@"; do
             OUTPUT_STYLE="$2"
             shift
             ;;
+        --author-commit-limit)
+            AUTHOR_COMMIT_LIMIT="$2"
```

Comment:
Do we need to check if the commit limit exists?

Reply:
Perhaps we should. I Just copied the code for `--output-style`.